src/features/account/components/profile-form.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { User as AuthUser } from "lucia";
+import { useActionState } from "react";
+import { FieldError } from "@/components/form/field-error";
+import { Form } from "@/components/form/form";
+import { SubmitButton } from "@/components/form/submit-button";
+import { EMPTY_ACTION_STATE } from "@/components/form/utils/to-action-state";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import { updateProfile } from "../actions/update-profile";
+
+type ProfileFormProps = {
+  user: AuthUser;
+};
+
+const ProfileForm = ({ user }: ProfileFormProps) => {
+  const [actionState, action] = useActionState(
+    updateProfile.bind(null),
```

Comment:
Replace `updateProfile.bind(null)` with `updateProfile` if you do not want to provide any extra arguments. See https://www.robinwieruch.de/react-form-server-action-extra-arguments/

Reply:
Yes sure, this issue happen when copy/pasting :)

Reply:
That's alright. I find this shows nicely that we have great building blocks in this course so that we can implement such forms pretty straigthfoward now!